ocs_ci/ocs/resources/mcg.py

```diff
@@ -351,7 +352,7 @@ def _retrieve_reduction_data():
         except TimeoutExpiredError:
             logger.error(
                 'Data reduction is insufficient. '
-                f'{total_size - total_reduced} bytes reduced out of {expected_reduction}.'
+                f'{total_size - total_reduced} bytes reduced out of {expected_reduction_in_bytes}.'
             )
             assert False
```

Comment:
Please also log the amount reduced vs. the amount expected in the assertion. This way we won't have to look for this info in the logs upon every failure